examples/hello_world/client/rust/src/main.rs

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Hello World client that sends 4 concurrent
+
+#![feature(async_closure)]
+
+use hello_world_grpc::proto::{hello_world_client::HelloWorldClient, HelloRequest};
+use log::info;
+use oak_abi::label::Label;
+use prost::Message;
+use structopt::StructOpt;
+use tonic::{
+    metadata::MetadataValue,
+    transport::{Certificate, Channel, ClientTlsConfig},
+    Request,
+};
+#[derive(StructOpt, Clone)]
+#[structopt(about = "Hello World Client")]
+pub struct Opt {
+    #[structopt(
+        long,
+        help = "URI of the Oak application to connect to",
+        default_value = "https://localhost:8080"
+    )]
+    uri: String,
+    #[structopt(
+        long,
+        help = "PEM encoded X.509 TLS root certificate file used by gRPC client"
+    )]
+    root_tls_certificate: String,
```

Comment:
If you called this option `--ca_cert`, would the need for `additional_args` in `example.toml` disappear?

Reply:
Do we pass --ca_cert to all examples?

Reply:
Not sure &ndash; [maybe just for Bazel-built clients?](https://github.com/project-oak/oak/blob/main/runner/src/main.rs#L823)

Reply:
Sooo many points! I will create another PR for this since I need to also change the TIR example. Tracking in #1336